crates/router/src/services/email/types.rs

```diff
@@ -76,7 +76,7 @@ pub mod html {
             EmailBody::MagicLink { link, user_name } => {
                 format!(
                     include_str!("assets/magic_link.html"),
-                    user_name = user_name,
+                    username = user_name,
```

Comment:
Why do we need this change ?

Reply:
To maintain the consistency, as in other emails, if you see, we use username for html files and user_name is what we are assigning to username. No other reason. Hence utilised "username" in the Magiclink.html file where it was required rather than user_name.